cookbooks/cdo-mysql/test/integration/readwrite/inspec/test_spec.rb

```diff
@@ -0,0 +1,48 @@
+def cmd(exec, match = nil, should_match = true)
+  return command(exec).stdout if match.nil?
```

Comment:
I think it'd improve clarity if this was broken into two separate functions, perhaps `def cmd(exec)` and `def cmd_assert(exec, match, should_match = true)`.